code/modules/cargo/bounties/assistant.dm

```diff
@@ -0,0 +1,254 @@
+/*
+/datum/bounty/item/assistant/strange_object
+	name = "Strange Object"
+	description = "[current_map.company_name] has taken an interest in strange objects. Find one in maint, and ship it off to [current_map.boss_short] right away."
+	reward = 1200
+	wanted_types = list(/obj/item/relic)
+
+/datum/bounty/item/assistant/scooter
+	name = "Scooter"
+	description = "[current_map.company_name] has determined walking to be wasteful. Ship a scooter to [current_map.boss_short] to speed operations up."
+	reward = 1080 // the mat hoffman
+	wanted_types = list(/obj/vehicle/ridden/scooter)
+	include_subtypes = FALSE
+
+/datum/bounty/item/assistant/skateboard
+	name = "Skateboard"
+	description = "[current_map.company_name] has determined walking to be a wasteful. Ship a skateboard to [current_map.boss_short] to speed operations up."
+	reward = 900 // the tony hawk
+	wanted_types = list(/obj/vehicle/ridden/scooter/skateboard)
+*/
+/datum/bounty/item/assistant/stunprod/New()
+	..()
+	name = "Stunprod"
+	description = "[current_map.boss_short] demands a stunprod to use against dissidents. Craft one, then ship it."
+	reward = 1300
+	wanted_types = list(/obj/item/weapon/melee/baton/cattleprod)
+
+/datum/bounty/item/assistant/soap/New()
+	..()
+	name = "Soap"
+	description = "Soap has gone missing from [current_map.boss_short]'s bathrooms and nobody knows who took it. Replace it and be the hero [current_map.boss_short] needs."
+	reward = 2000
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/soap)
+
+/datum/bounty/item/assistant/spear/New()
+	..()
+	name = "Spears"
+	description = "[current_map.boss_short]'s security forces are going through budget cuts. You will be paid if you ship a set of spears."
+	reward = 2000
+	required_count = 5
+	wanted_types = list(/obj/item/weapon/material/twohanded/spear)
+
+/datum/bounty/item/assistant/toolbox/New()
+	..()
+	name = "Toolboxes"
+	description = "There's an absence of robustness at [current_map.boss_name]. Hurry up and ship some toolboxes as a solution."
+	reward = 2000
+	required_count = 6
+	wanted_types = list(/obj/item/weapon/storage/toolbox)
+
+/datum/bounty/item/assistant/statue/New()
+	..()
+	name = "Statue"
+	description = "[current_map.boss_name] would like to commision an artsy statue for the lobby. Ship one out, when possible."
+	reward = 2000
+	wanted_types = list(/obj/structure/sculpting_block)
+
+/datum/bounty/item/assistant/statue/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/structure/sculpting_block/S = O
+	if(S && S.sculpted)
+		return TRUE
+	return FALSE
+
+/datum/bounty/item/assistant/cheesiehonkers/New()
+	..()
+	name = "Cheesie Honkers"
+	description = "Apparently the company that makes Cheesie Honkers is going out of business soon. [current_map.boss_short] wants to stock up before it happens!"
```

Comment:
This entire bounty isn't really that good tbh.

Reply:
Bumped the amount so it should be a slight challange